internal/rqlite/batching.go

```diff
@@ -0,0 +1,47 @@
+package rqlite
+
+import (
+	"context"
+	"errors"
+
+	"github.com/pbinitiative/zenbpm/internal/appcontext"
+	"github.com/pbinitiative/zenbpm/internal/log"
+	"github.com/rqlite/rqlite/v8/command/proto"
+)
+
+// Adds a statement to an executionKey's transaction
+func (persistence *PersistenceRqlite) addToTransaction(key int64, stmt *proto.Statement) {
```

Comment:
Can we add here or into the PersistenceRqlite struct a comment `// TODO: rework needed to support transactions without mutexes` ?